src/Treeview.ts

```diff
@@ -169,6 +170,12 @@ export class JsonOutlineProvider implements vscode.TreeDataProvider<string> {
             this.tree = Json.parse(this.text);
         }
     }
+    private getContextValue(elementInfo: IElementInfo): string {
```

Comment:
Please add these comments:
  /**
   * Retrieves the context value of the tree item, which can be used in the "where" clause of an `view/item/context`
   * menu, as `viewItem == <contextValue>`
   */

Reply:
Fixed